platform/android/CHANGELOG.md

```diff
@@ -25,7 +48,6 @@ This release changes how offline tile requests are billed — they are now bille
 
 ### Bug fixes
  - Fixed a rendering issue caused by all icons being treated as SDFs if an SDF and non-SDF icon were in the same layer. [#15456](https://github.com/mapbox/mapbox-gl-native/pull/15456)
- - Fixed an issue where changing location's render mode when the`LocationComponent` is disable wouldn't invalidate the foreground icon when it's back enabled. [#15507](https://github.com/mapbox/mapbox-gl-native/pull/15507)
```

Comment:
This was actually shipped with `8.3.0` and shouldn't be removed from here.

Reply:
Whoops. Re-added.